htdocs/product/stats/commande_fournisseur.php

```diff
@@ -300,7 +300,7 @@
 						print '<td class="center">'.$objp->qty."</td>\n";
 						print '<td align="right">'.price($objp->unitprice)."</td>\n";
 						print '<td align="right">'.price($objp->total_ht)."</td>\n";
-						print '<td align="right">'.$supplierorderstatic->getLibStatut(4).'</td>';
+						print '<td align="right">'.$supplierorderstatic->LibStatut($objp->statut, 4, $objp->billed).'</td>';
```

Comment:
I see no reason to have the new code working better than old one. Can you explain ?
$supplierorderstatic->getLibStatut(4)
is same than 
$supplierorderstatic->LibStatut($supplierorderstatic->status, 4, $supplierorderstatic->billed)
and $supplierorderstatic->status is set to $objp->status
and $supplierorderstatic->billed is set to $objp->billed